src/schema/users.ts

```diff
@@ -413,6 +413,16 @@ export const typeDefs = /* GraphQL */ `
     lastViewAt: DateTime
   }
 
+  enum AcquisitionChannel {
```

Comment:
There's also form_dismissed mentioned in RFC, do we want to add here?

Reply:
I think that was actually a mistake. I am not sure we want to record that, as it won't help. Thoughts? cc @dailydotdev/web-team

Reply:
Good point. Let's probably check with the product/analytics team whether they want to monitor the amount of users that ignored and dismissed. Wdyt?

Reply:
Has product requested this? Let's confirm with them better.

Reply:
Based on latest conversation, we will rely on tracking event for the analytics, so no need to store it in the DB.